src/app/components/login/login.blp

```diff
@@ -94,6 +94,13 @@ template $LoginWindow : Adw.Window {
             halign: center;
             styles ["pill", "suggested-action"]
           }
+
+          Button login_with_spotify_button {
+            /* Translators: Log in button label */
+            label: _("Log in with Spotify");
```

Comment:
the wording could probably be improved to make it clear its an external login, but I dont have any great suggestion just yet; probably need to review the whole ui of this dialog!

Reply:
Login with Spotify.com?